components/match2/wikis/splitgate/get_match_group_copy_paste_wiki.lua

```diff
@@ -0,0 +1,86 @@
+---
+-- @Liquipedia
+-- wiki=splitgate
+-- page=Module:GetMatchGroupCopyPaste/wiki
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Table = require('Module:Table')
+
+--[[
+
+WikiSpecific Code for MatchList, Bracket, and SingleMatch Code Generators
+
+]]--
+
+local wikiCopyPaste = Table.copy(require('Module:GetMatchGroupCopyPaste/wiki/Base'))
+
+--allowed opponent types on the wiki
+local MODES = {
+	['solo'] = 'solo',
+	['team'] = 'team',
+	['literal'] = 'literal',
```

Comment:
@hjpalpha Should literal be added to Base? Since base getOpponent has a case for literal, but not in the modes? If we add Literal in base, then this can be dropped here.

Reply:
agreed

Reply:
#885